src/unittest/extproc_test.cc

```diff
@@ -263,9 +263,9 @@ class segfault_job_t : public base_crash_job_t {
 
 private:
     static bool worker_fn(read_stream_t *, write_stream_t *) {
-        int bad_data = *reinterpret_cast<volatile int*>(NULL);
```

Comment:
Was causing a compilation error on clang. I do not have the slightest idea why, but it wasn't liking them for some reason.